content/300-guides/200-deployment/00-deploying-to-heroku.md

```diff
@@ -180,78 +185,47 @@ Database has been created and is available
-
-## 6. Set the DATABASE_URL environment variable locally
-
-In the previous step you created the database and saw how Heroku defines `DATABASE_URL` for the application when running on Heroku. In this step, you will set the `DATABASE_URL` environment variable locally so that you can run the database migration from locally using Prisma.
-
-Get the connection URL with the Heroku CLI:
-
-```no-lines
-heroku pg:credentials:url
-
-Connection information for default credential.
-Connection URL:
-   postgresql://__USER__:__PASSWORD__@__HOST__:__PORT__/__DATABASE__
-```
-
-Copy the connection URL and set it as an environment variable:
-
-```no-lines
-export DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__:__PORT__/__DATABASE__"
-```
-
-> **Note:** It's considered best practice to keep secrets out of your codebase. If you open up the `prisma/schema.prisma` file, you should see `env("DATABASE_URL")` in the _datasource_ block. By setting an environment variable you keep secrets out of the codebase.
-
-## 7. Create the database schema
-
-With the Heroku app and database created, you will create the database schema using the [`prisma db push`](/reference/api-reference/command-reference#db-push) command.
-
-
-> **Note:** While the `prisma db push` command creates the database schema for you. It's intended for quick prototyping. When working on a production project with multiple environments, Prisma Migrate gives you more control over how the database schema is created and evolved.
-
-Run the following command to create the database schema:
-
-```no-lines
-npx prisma db push --preview-feature
-```
-
-> **Note:** The `db push` command is currently in [Preview](/about/releases#preview). This means that it is not recommended to use in production.
-
-**Checkpoint:** `heroku pg:psql --command="\dt"` should show the newly created database tables:
-
-```no-lines
-  --> Connecting to postgresql-parallel-111222
-                List of relations
-  Schema |    Name    | Type  |     Owner
-  --------+------------+-------+----------------
-  public | Post       | table | alice
-  public | User       | table | alice
-  (2 rows)
-```
+> **Note:** Heroku automatically sets the `DATABASE_URL` environment variable when the app is running on Heroku. Prisma uses this environment variable because it's declared in the _datasource_ block of the Prisma schema (`prisma/schema.prisma`) with `env("DATABASE_URL")`.
 
-## 8. Push to deploy
+## 6. Push to deploy
 
 Deploy the app by pushing the changes to the Heroku app repository:
 
 ```no-lines
 git push heroku main
 ```
 
-This will trigger a build and deploy your application to Heroku.
+This will trigger a build and deploy your application to Heroku. Heroku will also run the `npx prisma migrate deploy` command which executes the migrations to create the database schema before deploying the app (as defined in the `release` step of the `Profile`).
```

Comment:
```suggestion
This will trigger a build and deploy your application to Heroku. Heroku will also run the `npx prisma migrate deploy` command which executes the migrations to create the database schema before deploying the app (as defined in the `release` step of the `Procfile`).
```